robyn/reloader.py

```diff
@@ -117,16 +115,16 @@ class EventHandler(FileSystemEventHandler):
     def __init__(self, file_path: str, directory_path: str) -> None:
         self.file_path = file_path
         self.directory_path = directory_path
-        self.process = None  # Keep track of the subprocess
+        self.process: subprocess.Popen[bytes] | None = None  # Keep track of the subprocess
```

Comment:
Hey @dave42w 👋 

Can you use `Union` here instead? We still have a python 3.8 ci right now. Yes, we should get rid of it soon but we have it running right now 😓